dynamic.js

```diff
@@ -218,6 +218,7 @@ module.exports = function (fastify, opts, next) {
       }
 
       swaggerMethod.responses = genResponse(schema ? schema.response : null)
+      /* istanbul ignore next */
```

Comment:
I've ignored the following check 'cause we'll always have a `swaggerRoute` in place, therefore, we have no need to take `else` path

Reply:
Then let's drop the if?

Reply:
I'll do so